source/draw/gpu/GpuMap.ooc

```diff
@@ -77,7 +79,9 @@ GpuMap: abstract class {
 	}
 	use: virtual func
 	add: func <T> (key: String, value: T) {
-		if (T inheritsFrom?(Object))
+		if (T inheritsFrom?(Cell))
+			Debug raise("Can't add Cell to GpuMap!")
```

Comment:
Because I assume all the cells in the map have been created internally and therefore should be freed by the map itself. A bit ugly but it doesn't make sense to upload a Cell to the GPU. 

I could improve this now however, after my latest change where I added the `_ownsObject` bool instead of just checking if the object is a `Cell` when freeing. So only when you add a value type and an internal Cell is created by `GpuMap` this bool is set to true. That would be a better solution.